homeassistant/components/vacuum/mqtt.py

```diff
@@ -0,0 +1,251 @@
+"""
+Support for a generic MQTT vacuum.
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/vacuum/
+"""
+import asyncio
+import json
+import logging
+
+import voluptuous as vol
+
+import homeassistant.components.mqtt as mqtt
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.vacuum import (
+    DEFAULT_ICON, SUPPORT_BATTERY, SUPPORT_CLEAN_SPOT,
+    SUPPORT_LOCATE, SUPPORT_PAUSE, SUPPORT_RETURN_HOME,
+    SUPPORT_STATUS, SUPPORT_STOP, SUPPORT_TURN_OFF,
+    SUPPORT_TURN_ON, VacuumDevice, services_to_strings, strings_to_services)
+from homeassistant.const import CONF_NAME, ATTR_SUPPORTED_FEATURES,\
```

Comment:
Please sort the imported names alphabetically :abcd: and break the line with parenthesis.

Reply:
Is there a doc somewhere about code-style things that aren't otherwise enforced by `flake8`/`pylint`/`pydocstyle`?

Reply:
Here are our recommendations: https://home-assistant.io/developers/development_guidelines/. Neither of my comments here are mentioned there though.